homeassistant/components/ecobee/climate.py

```diff
@@ -563,8 +580,24 @@ def extra_state_attributes(self) -> dict[str, Any] | None:
             ),
             "equipment_running": status,
             "fan_min_on_time": self.settings["fanMinOnTime"],
+            "available_sensors": self.remote_sensors,
+            "active_sensors": self.active_sensors_in_preset_mode,
         }
 
+    @property
+    def remote_sensors(self) -> list:
+        """Return the remote sensor names of the thermostat."""
+        try:
+            sensors_info = self.thermostat["remoteSensors"]
+        except KeyError:
+            sensors_info = []
```

Comment:
I think we can simplify this to just:
```suggestion
        sensors_info = self.thermostat.get("remoteSensors", [])
```